ci/jenkins/test.sh

```diff
@@ -813,6 +822,11 @@ function run_e2e {
     go mod edit -replace github.com/moby/spdystream=github.com/antoninbas/spdystream@v0.2.1 && go mod tidy
     if [[ $TESTBED_TYPE == "flexible-ipam" ]]; then
         go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider remote -timeout=100m --prometheus --antrea-ipam
+    elif [[ $TESTBED_TYPE == "kind" ]]; then
+        kubectl apply -f build/yamls/antrea-prometheus.yml
+        kubectl apply --context kind-${KIND_CLUSTER} -f build/yamls/antrea-prometheus-rbac.yml
+        ./hack/generate-manifest.sh | docker exec -i ${KIND_CLUSTER}-control-plane dd of=/root/antrea.yml
+        go test -v antrea.io/antrea/test/e2e --logs-export-dir `pwd`/antrea-test-logs --provider kind -timeout=100m --prometheus
```

Comment:
Safer to run kubectl to use the right context before running the tests.
```
kubectl config use-context kind-${KIND_CLUSTER}
```
Maybe you can run this before line 826, then you don't have to pass the context in each step.